README.md

```diff
@@ -168,11 +168,11 @@ raygunClient.user = function (req) {
 **Param**: *req*: the current request.
 **Returns**: The current user's identifier, or an object that describes the user.
 
-This will be transmitted with each message sent, and a count of affected users will appear on the dashboard in the error group view. If you return an email address, and the user has associated a Gravatar with it, their picture will be also displayed.
+This will be transmitted with each message sent, and a count of affected customers will appear on the dashboard in the error group view. If you return an email address, and the user has associated a Gravatar with it, their picture will be also displayed.
 
 If you return an object, it may have any of the following properties (only identifier is required):
 
-`identifier` is the user identifier. This will be used to uniquely identify the user within Raygun. This is the only required parameter, but is only required if you are using user tracking.
+`identifier` is the user identifier. This will be used to uniquely identify the user within Raygun. This is the only required parameter, but is only required if you are using Customers.
```

Comment:
"if you are using Customers" does not sound good - perhaps "if you are using the Customer experience monitoring feature" or "if you have enabled customer tracking"?

Reply:
I'll use customer tracking here as 'Customer Experience Monitoring' is a phrase only to be used within marketing collateral